.github/dependabot.yml

```
@@ -53,6 +53,9 @@ updates:
       - dependency-name: "System.Reflection.Emit"
       - dependency-name: "System.Reflection.Emit.Lightweight"
       ### End Datadog.Trace.csproj ignored dependencies
+
+      # Lock Microsoft.Build.Framework for widest compatibility when instrumenting builds
+      - dependency-name: "Microsoft.Build.Framework"
```

nit: I think this should be _before_ the "ignored dependencies" comment?
```suggestion

      # Lock Microsoft.Build.Framework for widest compatibility when instrumenting builds
      - dependency-name: "Microsoft.Build.Framework"
      ### End Datadog.Trace.csproj ignored dependencies    
```

---

Oh, I had the all the Datadog.Trace.csproj dependencies grouped together since those needed to be copy/pasted to Datadog.Trace.ClrProfiler.Managed.csproj as well, but this one is unique to the latter project

---

Ah, I see, my bad!